source/core/src/main/com/csse3200/game/dialogue/DialogBox.java

```diff
@@ -0,0 +1,42 @@
+package com.csse3200.game.dialogue;
+
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Dialog;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+
+/**
+ * Class used to represent the DialogBox displayed on screen
+ *
+ * The DialogBox will be called upon certain events triggers with NPCs and other mobs,
+ * illustrating information for game players to learn
+ *
+ * @author GPA Reapers (Team 11 UI(Stats))
+ */
+public class DialogBox extends Dialog {
```

Comment:
This class is currently deprecated (unused in this branch and replaced with ChatOverlay in other branches) so should be removed